lib/compiler.js

```diff
@@ -9,6 +9,21 @@ module.exports = function(options) {
         var includes = [];
         var resourcePath = mapcache.get(id);
         var processDependency = function(token) {
+            if (options.twigOptions.namespaces) {
```

Comment:
You would need to check that `options.twigOptions` exists before using `options.twigOptions.namespaces`. That will probably fix the failing tests.

```suggestion
            if (options.twigOptions && options.twigOptions.namespaces) {
```

Reply:
Thanks, @ilkkave!